storm-core/src/jvm/org/apache/storm/state/KeyValueState.java

```diff
@@ -45,4 +45,11 @@
      * @return the value or defaultValue if no mapping is found
      */
     V get(K key, V defaultValue);
+
+    /**
+     * Deletes the value mapped to the key, if there is any
+     *
+     * @param key   the key
+     */
+    void delete(K key);
```

Comment:
Could probably be changed to `V remove(K key)` along the lines of java.util.Map.